pkg/controller/constrainttemplate/constrainttemplate_controller_test.go

```diff
@@ -274,8 +275,15 @@ anyrule[}}}//invalid//rego
 			}
 			if ct.Name == "invalidrego" {
 				status := util.GetCTHAStatus(ct)
-				if len(status.Errors) != 1 {
-					return errors.New("InvalidRego template should contain 1 parse error")
+				if len(status.Errors) == 0 {
+					j, err := json.Marshal(status)
+					var s string
+					if err != nil {
+						s = err.Error()
+					} else {
+						s = string(j)
+					}
+					return fmt.Errorf("InvalidRego template should contain an error: %s", s)
```

Comment:
Nit: This failure message is a bit misleading since a JSON marshaling error would be reported here. I'm not familiar with this BDD framework but perhaps just halting this particular test case with `t.Fatal(err)` on L282 would be clearer.

Reply:
+1, fixed